service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/ManageLegalCounselControllerTest.java

```diff
@@ -0,0 +1,106 @@
+package uk.gov.hmcts.reform.fpl.controllers;
+
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.springframework.boot.test.autoconfigure.OverrideAutoConfiguration;
+import org.springframework.boot.test.autoconfigure.web.servlet.WebMvcTest;
+import org.springframework.boot.test.mock.mockito.MockBean;
+import uk.gov.hmcts.reform.ccd.client.model.AboutToStartOrSubmitCallbackResponse;
+import uk.gov.hmcts.reform.ccd.client.model.CaseDetails;
+import uk.gov.hmcts.reform.ccd.model.Organisation;
+import uk.gov.hmcts.reform.fpl.model.CaseData;
+import uk.gov.hmcts.reform.fpl.model.LegalCounsellor;
+import uk.gov.hmcts.reform.fpl.model.common.Element;
+import uk.gov.hmcts.reform.fpl.service.legalcounsel.ManageLegalCounselService;
+
+import java.util.List;
+
+import static java.util.Arrays.asList;
+import static java.util.Collections.emptyList;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.mockito.Mockito.verify;
+import static org.mockito.Mockito.when;
+import static uk.gov.hmcts.reform.fpl.utils.CoreCaseDataStoreLoader.populatedCaseDetails;
+import static uk.gov.hmcts.reform.fpl.utils.ElementUtils.element;
+
+@WebMvcTest(ManageOrdersController.class)
+@OverrideAutoConfiguration(enabled = true)
+class ManageLegalCounselControllerTest extends AbstractCallbackTest {
+
+    @MockBean
+    private ManageLegalCounselService manageLegalCounselService;
```

Comment:
I wouldn't mock this, as this doesn't directly connect to outside apis, so there must be something else lower level that can be mocked

Reply:
That's a fair point. I'll address that.

Reply:
Done!